src/app/static/src/js/components/reportLog/runningReportDetails.vue

```diff
@@ -62,8 +62,8 @@
                     </div>
                 </div>
                 <div id="report-logs" class="row pt-2">
-                    <div class="text-right col-sm-10">
-                        <textarea class="form-control bg-white"
+                    <div class="text-right col-12">
+                        <textarea class="form-control bg-white text-monospace" style="font-size: 80%;"
```

Comment:
* It just so happens that Bootstrap's `.small` class is 80%, so I guess you could use that instead - at the loss of some flexibility.
* Also, I know not part of this PR but you could expand `rows` to 20 - on my screen it feels like we have space to burn.